drivers/sdcard_spi/sdcard_spi.c

```diff
@@ -116,7 +116,7 @@ static sd_init_fsm_state_t _init_sd_fsm_step(sdcard_spi_t *card, sd_init_fsm_sta
 
         if (gpio_is_valid(card->params.power)) {
             gpio_write(card->params.power, card->params.power_act_high);
-            xtimer_usleep(SD_CARD_WAIT_AFTER_POWER_UP_US);
+            ztimer_sleep(ZTIMER_USEC, SD_CARD_WAIT_AFTER_POWER_UP_US);
```

Comment:
This could be converted to use `ZTIMER_MSEC` if you think that makes sense here.